annorxiver_modules/corpora_comparison_helper.py

```diff
@@ -115,86 +121,79 @@ def dump_to_dataframe(count_dict, file_name):
     This function outputs the counter data object in
     the form of a tsv file. This was created to avoid
     the memory issue PMC creates.
-    
+
     Arguments:
         count_dict - the counter object to be outputted
         file_name - the name of the file for the output to be written to
     """
 
     # Avoids memory issues especially with the PMC.
-    # A dictionary form of PMC tokens cannot be picked 
+    # A dictionary form of PMC tokens cannot be picked
     # without ram > 64GB.
-    # This solution prevents crashes and is quite convenient to 
+    # This solution prevents crashes and is quite convenient to
     # work with.
     with open(file_name, "w") as tsvfile:
         reader = csv.DictWriter(
-            tsvfile, 
-            fieldnames=["lemma", "pos_tag", "dep_tag", "count"], 
-            delimiter="\t"
+            tsvfile, fieldnames=["lemma", "pos_tag", "dep_tag", "count"], delimiter="\t"
         )
 
         reader.writeheader()
         for item in tqdm_notebook(count_dict):
-            reader.writerow({
-                "lemma":item[0],
-                "pos_tag":item[1],
-                "dep_tag":item[2],
-                "count":count_dict[item]
-            })
-
-def get_term_statistics(corpus_one, corpus_two, freq_num, psudeocount=1, disable_progressbar=False):
+            reader.writerow(
+                {
+                    "lemma": item[0],
+                    "pos_tag": item[1],
+                    "dep_tag": item[2],
+                    "count": count_dict[item],
+                }
+            )
+
+
+def get_term_statistics(
+    corpus_one, corpus_two, freq_num, psudeocount=1, disable_progressbar=False
+):
     """
     This function is designed to perform the folllowing calculations:
         - log likelihood of contingency table
         - log odds ratio
-        
+
     keywords:
-        corpus_one - a dataframe object with terms and counts 
+        corpus_one - a dataframe object with terms and counts
         corpus_two - a datafram object with terms and counts
         freq_num - number of most common words to use from both corpora
         psudeocount - the psudocount to avoid divide by zero
         disable_progressbar - show the progress bar?
     """
-    term_list = (
-        set(
-            corpus_one
-            .sort_values("count", ascending=False)
-            .head(freq_num)
-            .lemma
-            .values
-        )
-        |
-        set(
-            corpus_two
-            .sort_values("count", ascending=False)
-            .head(freq_num)
-            .lemma
-            .values
-        )
+    term_list = set(
+        corpus_one.sort_values("count", ascending=False).head(freq_num).lemma.values
+    ) | set(
```

Comment:
cool didn't know you could use this shorthand operator!

Reply:
Yeah its great to utilize!